app/views/cloud_volume/new.html.haml

```diff
@@ -11,5 +11,5 @@
 
 :javascript
   ManageIQ.angular.app.value('cloudVolumeFormId', '#{@volume.id || "new"}');
-  ManageIQ.angular.app.value('storageManagerId', '#{@storage_manager.try(:id)}');
+  ManageIQ.angular.app.value('storageManagerId', #{@storage_manager.try(:id) || "undefined"});
```

Comment:
Here we render JavaScript with either `('storageManagerId', 23)` (23 being the ID of the storage manager) or `('storageManagerId', undefined)` (in case `storage_manager.nil?`). I think this is quite clean way of letting JS controller now that there's nothing to do.